blaze/expr/collections.py

```diff
@@ -31,19 +33,200 @@
            'isin',
            'Join',
            'join',
+           'Key',
            'Merge',
            'merge',
            'Sample',
            'sample',
            'Shift',
            'shift',
            'Sort',
+           'Sort2',
            'sort',
+           'sort2',
            'Tail',
            'tail',
            'transform']
 
 
+class Key(Expr):
+
+    """Expression sorted in ascending or descending order"""
+
+    _arguments = '_child', 'table', '_key', 'ascending'
+
+    def _dshape(self):
+        return self._child.dshape
+
+    @property
+    def key(self):
```

Comment:
Some test case expected a list instead of a tuple.  Multiple keys are again a leftover from ```data.asc() == data.asc(keys=['a', 'b', ...])```.